docs/LoongArch-ELF-ABI-EN.adoc

```diff
@@ -252,71 +256,66 @@ can be found in http://sco.com/developers/gabi/latest/contents.html[the latest S
 |ELF64 object file
 |===
 
-=== `e_machine`: Identifies the machine
+=== e_machine: Identifies the machine
 
 `LoongArch (258)`
 
-=== `e_flags`: Identifies ABI type and version
+=== e_flags: Identifies ABI type and version
 
 [%header,cols="1,1,1,1"]
 |====
 | Bit 31 - 8 | Bit 7 - 6   | Bit 5 - 3      | Bit 2 - 0
-| (reserved) | ABI version | ABI extension  | Base ABI
+| (reserved) | ABI version | ABI extension  | Base ABI (partial)
```

Comment:
I think we should give bits[2-0] a better name. Something like 'FP ABI Maker' as @xen0n proposed can be considered?